src/components/ButtonPanel.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import Button from './Button';
+
+class ButtonPanel extends React.Component {
+  // eslint-disable-next-line class-methods-use-this
+  renderButtons(i, c, d) {
+    return (
+      <Button
+        name={i}
+        color={c}
+        wide={d}
+      />
+    );
+  }
```

Comment:
Nice! 

OPTIONAL: I would encourage you to have more meaningful names than `i`, `c`, and `d` for your arguments. It really makes difficult to read your code.